### PR TITLE
Fabo/fix app e2e

### PR DESCRIPTION
**Pull request by faboweb**

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5ede6b0e43ebacd89939ae0c--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
> 🚀 Deployed on https://5ede7b96c9be2775a6177836--lunieio.netlify.app

Is this a bot? How did you do it?

I want to learn how you did all this stuff. GH Actions?

**Comment by faboweb** [comment]
> I want to learn how you did all this stuff. GH Actions?

let's do on Thursday

**Comment by Bitcoinera** [comment]
> let's do on Thursday

Yes, knowledge passing! 💃 🔥

File: app/tests/e2e/helpers.js [hunk 1]
```diff
@@ -76,7 +80,7 @@ async function waitForText(
 }
 
 async function getLastActivityItemHash(browser) {
-  await browser.waitForElementVisible(".tx-container .tx", 10000)
+  await browser.waitForElementVisible(".tx-container .tx", 50000)
```
Comment (Bitcoinera) [inline app/tests/e2e/helpers.js:83]: Maybe my timeout is too high or is it OK?

Comment (Bitcoinera) [reply app/tests/e2e/helpers.js:83]: Well, check passed, so this looks good

File: app/tests/e2e/helpers.js [hunk 2]
```diff
@@ -227,21 +220,15 @@ async function actionModalCheckout(
...
-  return await browser.url(
-    browser.launch_url + browser.globals.slug + "/portfolio",
-    async () => {
-      // waiting till balance loaded
-      await browser.waitForElementVisible(".total", 5000, false)
-      const { value: total } = await browser.getText(".total")
-      browser.globals.denom = total.split(" ")[1]
-      browser.globals.totalAtoms = total.split(" ")[0]
+  await browser.url(browser.launch_url + browser.globals.slug + "/portfolio")
+  const denom = await getDenom(browser)
```
Comment (Bitcoinera) [inline app/tests/e2e/helpers.js:224]: oh, much cleaner 👍 !

File: app/tests/e2e/nightwatchGlobals.js [hunk 3]
```diff
@@ -305,40 +272,7 @@ async function switchToAccount(
...
-        continue
-      }
-      apiUp = true
-    } catch (err) {
-      console.log("Failed to check API", err)
-      console.log("Waiting 1s for API to be up")
-      await new Promise((resolve) => setTimeout(resolve, 1000))
-    }
-  }
-}
```
Comment (Bitcoinera) [inline app/tests/e2e/nightwatchGlobals.js:344]: Why do we remove this check? Well, if API is not up it fails anyway no? But will we know why? This should help us identify the reason no

Comment (faboweb) [reply app/tests/e2e/nightwatchGlobals.js:344]: I tried to simplify and speed up. E2E tests use the deployed apis which are always up. This is only used for localhost.

Comment (Bitcoinera) [reply app/tests/e2e/nightwatchGlobals.js:344]: Cool, thank you for explanation

File: app/tests/e2e/helpers.js (hunk 1 above)
Comment (faboweb) [reply app/tests/e2e/helpers.js:83]: I forgot this. The timout might be too high as the global timeout is only 30000

**Pull request merged**